citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/AuthorityServiceHelper.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.citesphere.core.service.impl;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthorityServiceHelper {
+
+    @Value("${_conceptpower_authority_search_keyword}")
+    private String conceptpowerSearchKeyword;
+
+    public String getConceptpowerSearchString(String firstName, String lastName) {
```

Comment:
this should probably also be part of the conceptpower importer (or at least it should be called from there and be specific to conceptpower).